platform_plugin_turnitin/tasks.py

```diff
@@ -0,0 +1,96 @@
+"""This module contains the tasks that will be run by celery."""
+
+import tempfile
+from urllib.parse import urljoin
+
+import requests
+from celery import shared_task
+from django.conf import settings
+from rest_framework import status
+from submissions import api as submissions_api
+
+from platform_plugin_turnitin.api.v1.views import TurnitinClient
+from platform_plugin_turnitin.constants import ALLOWED_FILE_EXTENSIONS
+from platform_plugin_turnitin.edxapp_wrapper import user_by_anonymous_id
+
+
+@shared_task
+def ora_submission_created_task(submission_id: str, file_downloads: list) -> None:
+    """
+    Task to handle the creation of a new ora submission.
+
+    Args:
+        submission_id (str): The ORA submission ID.
+        file_downloads (list): The list of file downloads.
+    """
+    submission_data = dict(submissions_api.get_submission_and_student(submission_id))
+    user = user_by_anonymous_id(submission_data["student_item"]["student_id"])
+
+    send_text_to_turnitin(submission_id, user, submission_data["answer"])
+    send_uploaded_files_to_turnitin(submission_id, user, file_downloads)
```

Comment:
So, these two submissions/answers are two different submissions in Turnitin?

Reply:
Yes, they are different submissions in Turnitin.
A submission is created for the text response, and each file as well. But they are all related to the same ORA submission ID.